packages/experimental/Link/src/Link.types.ts

```diff
@@ -7,6 +7,7 @@ export const linkName = 'Link';
 export type LinkState = IPressableState & {
   /**
    * Specifies whether the link has been visited.
+   * Note: Not supported for Android
```

Comment:
Please update similar comments in SPEC.md

Reply:
Seems in the experimental folder we don't have a SPEC.md for Link.
But the SPEC.md is present in the components folder

Reply:
Yes, I moved the SPEC file there to prepare LinkV1 for release.

Reply:
Once this is fixed feel free to merge

Reply:
Updated spec.md.